### DISCUSSION TITLE
Open feed items with custom app other than the browser?

### DISCUSSION THREAD

**Discussion by Mr-Bajs** [spacecowboy/Feeder#223] [Ideas]
Feeder is excellent.
It would be even greater if some feed items could be open with another app you have installed.

The use case I think of is if you subscribe on a youtube channel through Feeder, it would be awesome if the youtube.com feeds where opened with tha app of your choice rather than the browser, example it opens up youtube feed through [Tubular](https://github.com/TeamNewPipe/NewPipe) or [NewPipe](https://github.com/TeamNewPipe/NewPipe).

It's mainly videohosting sites I think of but i suppose audio like podcasts also would be nice.

**Comment by spacecowboy** [comment]
Opening the feeds with another app is impossible.

But if you actually mean *articles*, then opening in a custom app is not different than a browser. But you need to configure the feed to open with `Default browser`.

All Feeder does is send a generic intent with the URI:

```
Intent(Intent.ACTION_VIEW, Uri.parse(link))
```

If any app on you device is the default for this, then they will open it. Or the system might give you a popup to choose a particular app if several register that they handle that intent for that particular type of URI.

**Reply by Mr-Bajs** [reply]
Okey. So wouldn't it be possible to create custom rule to open links within Feeder from m.youtube.com or youtube.com with a certain app of your own choice?

**Reply by spacecowboy** [reply]
technically yes.
